tierkreis_visualization/pyproject.toml

```diff
@@ -21,4 +21,5 @@ tierkreis = { workspace = true }
 
 [project.scripts]
 tkr-vis = "tierkreis_visualization.main:start"
-
+tkr_viz_dev = "tierkreis_visualization.main:dev"
```

Comment:
`tkr-viz` versus `tkr_viz` might be a bit confusing. Though I think possibly these scripts might make more sense as subcommands of the main `tkr` CLI anyway in future?